jtlibrary/python/jtlibrary/src/jtlib/features.py

```diff
@@ -437,8 +437,9 @@ class Texture(Features):
     '''
 
     def __init__(self, label_image, intensity_image,
-            theta_range=4, frequencies={1, 5, 10}, radius={1, 5, 10},
-            threshold=None, compute_haralick=False):
+                 theta_range=4, frequencies=[1, 5, 10], radius=[1, 5, 10],
```

Comment:
I don't remember why I changed set to list. It may have been for compatibility/consistency with the handles input, where the only possibilities to pass multiple parameter values is a "Sequence" (as far as I know). In principle set is better but would need to be tested